### DISCUSSION TITLE
Issues with recipes and sprinkles while testing with Jest in a NextJS 13 project

### DISCUSSION THREAD

**Discussion by pxlmvr** [vanilla-extract-css/vanilla-extract#1070] [Q&A]
Hello,

I'm new to vanilla-extract. I have a Next JS 13 project where I use `recipes` and `sprinkles` for styling. However I'm having troubles with running unit tests with Jest: when a component uses a recipe, the test fails because of an `Uncaught [TypeError: (0 , _stylesCss.baz) is not a function]` error (where `baz` is the name of the recipe). Similarly when the component uses sprinkles I get an `Uncaught [TypeError: (0 , _sprinklesCss.sprinkles) is not a function]` error.

I tried using the Jest transformer as per documentation but it didn't solve the issue. Disabling runtime styles didn't either.

I suspect this could depend on my Next app using SWC as a compiler instead of Babel, as apparently it automatically sets up its own transformers for Jest, but I'm not sure.

Does anybody know how to solve this kind of issue? Or are `recipes` and `sprinkles` incompatible with a test environment using SWC?

**Comment by janbiasi** [comment]
I've just opened an issue for this case, see https://github.com/vanilla-extract-css/vanilla-extract/issues/1131
